sql_generators/funnels/configs/ios_onboarding_main.toml

```diff
@@ -264,15 +324,15 @@ from_expression = """
   LEFT JOIN `moz-fx-data-shared-prod`.firefox_ios_derived.funnel_retention_clients_week_2_v1 r2 -- retained_week_2 is not included in funnel_retention_clients_week_4_v1
   ON ic.client_id = r2.client_id
   LEFT JOIN
-    (SELECT * FROM firefox_ios.events_unnested eu WHERE DATE(submission_timestamp) = @submission_date) eu
-  ON ic.client_id = eu.client_info.client_id
+    (SELECT * FROM firefox_ios.events_stream eu WHERE DATE(submission_timestamp) = @submission_date) eu
+  ON ic.client_id = eu.client_id
   LEFT JOIN
-    (SELECT client_info.client_id,
-    ANY_VALUE(`mozfun.map.get_key`(event_extra, 'sequence_id')) AS funnel_id,
-    1 + LENGTH(ANY_VALUE(`mozfun.map.get_key`(event_extra, 'sequence_id'))) - 
-             LENGTH(REPLACE(ANY_VALUE(`mozfun.map.get_key`(event_extra, 'sequence_id')), '_', '')) AS number_of_onboarding_cards
-    FROM firefox_ios.events_unnested
-    WHERE `mozfun.map.get_key`(event_extra, 'sequence_id') IS NOT NULL
+    (SELECT client_id,
+    ANY_VALUE(JSON_VALUE(event_extra.sequence_id)) AS funnel_id,
+    1 + LENGTH(ANY_VALUE(JSON_VALUE(event_extra.sequence_id))) - 
+             LENGTH(REPLACE(ANY_VALUE(JSON_VALUE(event_extra.sequence_id)), '_', '')) AS number_of_onboarding_cards
+    FROM firefox_ios.events_stream
```

Comment:
I made a minor modification to use the `events_stream` table here. This seems to reduce the query complexity by quite a bit. Let me know if this ok, it should be equivalent to `events_unnested` except that `events_stream` already stores unnested records

Reply:
Yes that looks good to me